src/pybroker/portfolio.py

```diff
@@ -344,12 +344,22 @@ def __init__(
         self._entry_id: int = 0
         self._trade_id: int = 0
 
-    def _calculate_fees(self, fill_price: Decimal, shares: Decimal) -> Decimal:
+    def _calculate_fees(self, fill_price: Decimal, shares: Decimal, type: Literal["buy", "sell"]) -> Decimal:
         fees = Decimal()
         if self._fee_mode is None or self._fee_amount is None:
             return fees
         if self._fee_mode == FeeMode.ORDER_PERCENT:
             fees = self._fee_amount / Decimal(100) * fill_price * shares
+        elif self._fee_mode == FeeMode.C_ORDER_PERCENT:
+            if type == "buy":
```

Comment:
The if/else is redundant here. You can remove the repeated section to make more readable:

fees = self._fee_amount / Decimal(100) * fill_price * shares
if fees < Decimal(5.0):
     fees = Decimal(5.0)
if type == "sell":
   fees = fees + Decimal(0.0001) * fill_price * shares